README.rst

```diff
@@ -43,6 +43,16 @@ Use the `traffic` subcommand to change the stacks traffic:
 
 For see more options use `lizzy traffic --help`.
 
+Change stack scale
+--------------------
+Use the `traffic` subcommand to change the stacks traffic:
```

Comment:
Please add note that this will always update multiple stacks since the behaviour is different in senza.